File: Cpp/Array/Given an array which consists of only 0, 1 and 2. Sort the array without using any sorting algo.cpp
void sort012(int a[], int n)
    {
        // coode here 
        int st=0,end=n-1,mid=0;
        while(mid<=end){
            if(a[mid]==0){
                swap(a[mid],a[st]);
                st++;
            }
            else if(a[mid]==2){
                swap(a[mid],a[end]);
                end--;
                mid--;
            }
            mid++;
        }
    }

File: Python/Array/Given an array which consists of only 0, 1 and 2. Sort the array without using any sorting algo.py
    def sort012(self,a,n):
        # code here
        st,end,mid=0,n-1,0
        while(mid<=end):
            if a[mid]==0:
                a[mid],a[st]=a[st],a[mid]
                st+=1
            elif a[mid]==2:
                a[mid],a[end]=a[end],a[mid]
                end-=1
                mid-=1
            mid+=1